{"additions": 1, "auther_ref": "pA_autcondUp_80X", "auther_sha": "2d8c5860ed6164a4201a2bc7970046dd6905084f", "author": "mandrenguyen", "body": "Update to latest GT with updated reco BS. Needed for self-consistency of relval wf 281 after #16787 \r\nUnfortunately, this GT crashes wf 281, seemingly due to an update to the L1 menu.  @mmusich can you please advise?  The error is as follows. \r\n\r\n%MSG-e GlobalObjectMapRecord:  HLTL1TSeed:hltL1sCastorMediumJetBptxAND  10-Dec-2016 10:29:47 CET Run: 1 Event: 1\r\n\r\n\r\n  ERROR: The requested algorithm name = L1_CastorMediumJet_BptxAND\r\n  does not exists in the trigger menu.\r\n  Returning zero pointer for getObjectMap\r\n\r\n\r\n%MSG\r\n----- Begin Fatal Exception 10-Dec-2016 10:29:47 CET-----------------------\r\nAn exception of category 'FailModule' occurred while\r\n   [0] Processing run: 1 lumi: 1 event: 1\r\n   [1] Running path 'HLT_PAL1CastorMediumJet_BptxAND_v1'\r\n   [2] Calling event method for module HLTL1TSeed/'hltL1sCastorMediumJetBptxAND'\r\nException Message:\r\n\r\nAlgorithm L1_CastorMediumJet_BptxAND, requested as seed by a HLT path, cannot be matched to a L1 algo name in any GlobalObjectMap\r\nPlease check if algorithm L1_CastorMediumJet_BptxAND is present in the L1 menu", "branch": "CMSSW_8_0_X", "changed_files": 1, "closed_at": "1481539504", "comments": 10, "commits": 1, "created_at": "1481359148", "deletions": 1, "labels": ["alca-approved", "comparison-available", "fully-signed", "orp-pending", "tests-approved"], "milestone": "CMSSW_8_0_X", "number": 16962, "release-notes": [], "review_comments": 0, "state": "closed", "title": "updating pA autoconditions for 80X", "updated_at": "1481539504", "user": "mandrenguyen"}